content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst

```diff
@@ -1,95 +1,82 @@
+click :guilabel:`Add a Line` to specify the various components that make up the production of the
+final product and their respective quantities. New components can be created on the fly through the
+BoM, or can be created beforehand in :menuselection:`Manufacturing --> Products --> Products
+--> Create`.
 
+.. image:: bill_configuration/bom-form.png
+   :align: center
+   :alt: Set up a Bill of Materials.
 
-.. image:: bill_configuration/bom_1.png
-    :align: center
+Specify a BoM for a product variant
+-----------------------------------
 
-.. warning::
-   The destination location should **not** be a scrap location. A scrap location is where you put
-   products that you don't need.
+BoMs can also be assigned to specific *Product Variants*. There are two ways to set up BoMs for
+product variants, after the product variant attributes have been configured on the product form.
 
-Using the same BoM to describe Variants
----------------------------------------
+The first method is to create one BoM per variant by creating a new BoM and specifying the
+:guilabel:`Product Variant`. The second method is to create one master BoM that contains all of the
+components, and specify which variant each component applies to using the :guilabel:`Apply on
+Variants` column.
 
-As suggested above, you can use *BOMs* for specific *Product Variants*. 
-Once the various attributes have been configured on the product form, there 
-are two ways to configure the appropriate BoM for the respective product combination. 
+.. image:: bill_configuration/bom-variants.png
+   :align: center
+   :alt: Product Variants in the Bill of Materials. 
 
-Either create one BoM per variant, by specifying the Product Variant
-in the dedicated field below the product name. Or use one BOM, that contains
-all of the components and for each component, indicate which variant it applies to, 
-using the *Apply on Variants* column, as shown below. 
+Operations
+==========
 
-.. image:: bill_configuration/bom_2.png
-    :align: center
+Add an :guilabel:`Operation` to a BoM to specify instructions for production and register time spent on an operation. To use this feature, first enable the :guilabel:`Work Orders` feature in
+:menuselection:`Manufacturing --> Configuration --> Settings --> Operations`.
 
-
-Adding Operations
-=================
-
-You can also add operations to your *BoM*, if you want workers to follow instructions
-or register time spent. To use this feature, enable the *Work Orders* feature in the 
-*Manufacturing* app settings, as shown below.
-
-.. image:: bill_configuration/bom_3.png
-    :align: center
+Then, when creating a new BoM, click on the :guilabel:`Operations` tab and click :guilabel:`Add a
+line` to add a new operation. In the :guilabel:`Create Operations` box, give the operation a name,
+specify the :guilabel:`Work Center` and duration settings. Like components, and specify a product
```

Comment:
Replace `and specify a product` with `Odoo gives the option to specify a product`